scripts/fetchSbomData.ts

```diff
@@ -0,0 +1,62 @@
+import fs from "fs";
+import fetch from "node-fetch";
+
+interface SBOMData {
+  dependencies?: Record<string, unknown>;
+  [key: string]: unknown; // Add more specific fields if known
+}
+
+interface FetchError extends Error {
+  response?: Response;
+}
+
+const fetchSBOMData = async (url: string): Promise<SBOMData> => {
+  const response = await fetch(url, {
+    headers: {
+      Accept: "application/vnd.github+json",
+      "X-GitHub-Api-Version": "2022-11-28",
+    },
+  });
+
+  if (!response.ok) {
+    const error: FetchError = new Error(`Error fetching SBOM data from ${url}`);
+    error.response = response;
+    throw error;
+  }
+
+  return (await response.json()) as SBOMData;
+};
+
+const fetchData = async (): Promise<void> => {
+  const feUrl =
+    "https://api.github.com/repos/ohcnetwork/care_fe/dependency-graph/sbom";
+  const beUrl =
+    "https://api.github.com/repos/ohcnetwork/care/dependency-graph/sbom";
```

Comment:
nitpick: let's store this globally as constants instead of keeping it inside the function.